Library/Formula/raqm.rb

```diff
@@ -0,0 +1,24 @@
+class Raqm < Formula
+  desc "Library for complex text layout"
+  homepage "https://github.com/HOST-Oman/libraqm"
+  url "https://github.com/HOST-Oman/libraqm/releases/download/v0.1.0/raqm-0.1.0.tar.gz"
+  sha256 "fbccf176ae0ff5aeb73ea3d7a0f2173c432d87a495c10ca2ecdf82323eaa5355"
+
+  depends_on "freetype"
+  depends_on "harfbuzz"
+  depends_on "fribidi"
+  depends_on "glib" => :recommended # for the test suite
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+    system "make", "check"
+  end
+
+  # Raqm does not install any programs
+  test do
+    system "true"
```

Comment:
It’s simpler to hardcode the flags because you can’t rely on the presence of `pkg-config` at test time: it’s only a *build* dependency. You should be able to do something similar to this (adapted from `tinyxml2.rb`, I haven’t tested it here):
```rb
system ENV.cc, "test.c", "-L#{lib}", "-lraqm", "-o", "test"
```

Feel free to comment here if you have any question regarding Ruby, or anything else that can help you :smiley: